docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-jumpcloud.md

```diff
@@ -0,0 +1,192 @@
+```
+
+For more information on host authentication in YugabyteDB using `ysql_hba_conf_csv`, refer to [Host-based authentication](../../../../secure/authentication/host-based-authentication/).
+
+#### ysql_ident_conf_csv
+
+This flag is used to add translation regex rules that map token claim values to PostgreSQL roles. The flag settings are used as records in the `yb_ident.conf` file as user-name maps. This file is used identically to `pg_ident.conf` to map external identities to database users. For more information, refer to [User name maps](https://www.postgresql.org/docs/11/auth-username-maps.html) in the PostgreSQL documentation.
+
+The following illustration shows an example flag configuration:
+
+![Configuring ysql_ident_conf_csv flag for OIDC](/images/yp/security/oidc-jumpcloud-identconf.png)
+
+The following are examples of possible rules:
+
+- Map a single user
+
+  ```sh
+  map1 user@yugabyte.com user
+  ```
+
+- Map multiple users
+
+  ```sh
+  map2 /^(.*)@devyugabyte\.com$ \1
+  ```
+
+- Map Roles <-> Users
+
+  ```sh
+  map1 OIDC.Test.Read read_only_user
+  ```
+
+#### yb.security.oidc_feature_enhancements
+
+This flag must be enabled to expose the OIDC functionality in Yugabyte Anywhere. Use the following API to set values for this flag.
+
+```sh
+curl -k --location --request PUT '<server-address>/api/v1/customers/<customerUUID>/runtime_config/00000000-0000-0000-0000-000000000000/key/yb.security.oidc_feature_enhancements' \
+--header 'Content-Type: text/plain' \
+--header 'Accept: application/json' \
+--header 'X-AUTH-YW-API-TOKEN: <api-token>' \
+--data 'true'
+```
+
+## Manage users and roles
+
+After OIDC-based authentication is configured, an administrator can manage users as follows:
+
+- In the universe, add database users or roles.
+
+  You need to add the users and roles that will be used to authenticate to the database. The role must be assigned the appropriate permissions in advance. Users will use their database user/role as their username credential along with their JWT as the password when connecting to the universe.
+
+  For information on managing users and roles in YugabyteDB, see [Manage users and roles](../../../../secure/authorization/create-roles/).
+
+- In YugabyteDB Anywhere, create YBA users.
+
+  Create a user in YugabyteDB Anywhere for each user who wishes to sign in to YBA to obtain their JWT.
+
+  To view their JWT, YBA users can sign in to YugabyteDB Anywhere, click the **User** icon at the top right, select **User Profile**, and click **Fetch OIDC Token**.
+
+  This is not required if you enabled the **Display JWT token on login** option in the YBA OIDC configuration, as any database user can copy the JWT from the YBA landing page without signing in to YBA.
+
+  For information on how to add YBA users, see [Create, modify, and delete users](../../../administer-yugabyte-platform/anywhere-rbac/#create-modify-and-delete-users).
```

Comment:
What about "Using your JWT"? doesn't that also apply?
Should these sections be done with a include?